nautobot/cloud/forms.py

```diff
@@ -0,0 +1,64 @@
+from django import forms
+
+from nautobot.cloud.models import CloudAccount
+from nautobot.core.constants import CHARFIELD_MAX_LENGTH
+from nautobot.core.forms import DynamicModelChoiceField, DynamicModelMultipleChoiceField, MultiValueCharField
+from nautobot.dcim.models import Manufacturer
+from nautobot.extras.forms import NautobotBulkEditForm, NautobotFilterForm, NautobotModelForm, TagsBulkEditFormMixin
+from nautobot.extras.models import SecretsGroup
+
+#
+# Cloud Account
+#
+
+
+class CloudAccountForm(NautobotModelForm):
+    provider = DynamicModelChoiceField(
+        queryset=Manufacturer.objects.all(),
+        help_text="Manufacturers are the recommended model to represent cloud providers.",
+    )
+    secrets_group = DynamicModelChoiceField(queryset=SecretsGroup.objects.all())
+
+    class Meta:
+        model = CloudAccount
+        fields = [
+            "account_number",
+            "name",
+            "description",
+            "provider",
+            "secrets_group",
+            "tags",
+        ]
+
+
+class CloudAccountBulkEditForm(TagsBulkEditFormMixin, NautobotBulkEditForm):
+    pk = forms.ModelMultipleChoiceField(queryset=CloudAccount.objects.all(), widget=forms.MultipleHiddenInput)
+    secrets_group = DynamicModelChoiceField(queryset=SecretsGroup.objects.all(), required=False)
+    provider = DynamicModelChoiceField(queryset=Manufacturer.objects.all(), required=False)
+    description = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+
+    class Meta:
+        nullable_fields = [
+            "secrets_group",
+            "description",
+        ]
+
+
+class CloudAccountFilterForm(NautobotFilterForm):
+    model = CloudAccount
+    field_order = [
+        "q",
+        "account_number",
+        "name",
+        "provider",
+        "secrets_group",
+    ]
+    q = forms.CharField(required=False, label="Search")
+    name = MultiValueCharField(required=False, label="Name")
+    account_number = MultiValueCharField(required=False, label="Account Number")
```

Comment:
Are the `label` strings unnecessary/redundant for these two?

Reply:
Yeah they are redundant, removed.